stand/man/loader.efi.8

```diff
@@ -393,10 +393,10 @@ Boot to FW : false
 BootCurrent: 0001
 Timeout    : 2 seconds
 BootOrder  : 0000, 0001, 0003, 0004, 0005, 0006, 0001, 0008, 000A, 000B, 000C, 000E, 0007
-...
+\&...
 +Boot0001* FreeBSD ZPOOL HD(1,GPT,b5d0f86b-265d-1e1b-18aa-0ed55e1e73bd,0x28,0x96000)/File(\EFI\FREEBSD\LOADER.EFI)
```

Comment:
man interprets `\E`, `\F`, and `\L` as special commands.  I have a fix waiting for another PR; I don't think it belongs in this one since the commit says that they were reported by `mandoc`.

If you'd rather that I include the fixes now, I could make the message "Most of these were reported by `mandoc...`".

Reply:
I'll take what is here now and you can submit the other fix later :+1: